src/bun.js/bindings/CommonJSModuleRecord.cpp

```diff
@@ -494,15 +524,87 @@ JSC_DEFINE_HOST_FUNCTION(functionCommonJSModuleRecord_compile, (JSGlobalObject *
     return JSValue::encode(jsUndefined());
 }
 
+extern "C" JSC::EncodedJSValue jsFunctionResolveSyncPrivate(JSC::JSGlobalObject* lexicalGlobalObject, JSC::CallFrame* callFrame)
```

Comment:
☝️ Moved from ImportMetaObject to CommonJSModuleRecord because this is used in more contexts than the ImportMetaObject